### ISSUE TITLE
accessibility in the build pipeline

### ISSUE THREAD

**Issue by kategreenUSDS** [usds/website#763]
* audit
* verify content 

todo: what else would be contained in this?

**Comment by louiseClarkUSDS** [comment]
This will be determined once we have an solid understanding of our immediate tech stack versus our longer term tech stack. I did include information about including accessibility in the build process in the [A11y developer best practices Slack post](https://usds.enterprise.slack.com/files/TD42BGDLM/F04LC49QH52). @kategreenUSDS @rachellanman We can discuss this week how we might include some baseline automation testing.

**Comment by louiseClarkUSDS** [comment]
Right now we have Cypress-axe testing integrated. This is not designed to break the build -- it requires developers to manually run the cypress script to check the tests. The next Discovery Phase can look at more advanced strategies around automated testing.
